tests/bug-5220/bug-5220-6/test.yaml

```diff
@@ -0,0 +1,8 @@
+requires:
+  pcap: false
+  min-version: 7
+
+args:
+  - --engine-analysis
+
+exit-code: 0
```

Comment:
Should we test the output for what is used as fast pattern in the end ?

Reply:
Will add that. Thank you!